src/main.rs

```diff
@@ -58,34 +60,25 @@ fn main() {
     let mut pixels = {
         let window_size = window.inner_size();
         let surface_texture = SurfaceTexture::new(window_size.width, window_size.height, &window);
-        Pixels::new(DIMS.0, DIMS.1, surface_texture).expect("failed to create pixels")
+        PixelsBuilder::new(DIMS.0, DIMS.1, surface_texture)
+            .enable_vsync(true)
+            .build()
+            .expect("failed to create pixels")
     };
-
     // Fill alpha channel to avoid setting it later
     for pixel in pixels.get_frame_mut().chunks_exact_mut(4) {
         pixel[3] = 255u8;
     }
```

Comment:
We could do this here instead. It would set the other channels as well, but they just get overwritten. What do you think?
```suggestion
    pixels.get_frame_mut().fill(255);
```

Reply:
One could do this yes, and it doesn't cause any flickering because .render isn't called, so that would work.